projects/ad4170_iio/app/ad4170_iio.c

```diff
@@ -502,6 +536,51 @@ static int set_demo_config(void *device,
 	return -EINVAL;
 }
 
+/*!
+ * @brief Getter/Setter for the Reconfigure system attribute
+ * @param device[in]- pointer to IIO device structure
+ * @param buf[in]- pointer to buffer holding attribute value
+ * @param len[in] - length of buffer string data
+ * @param channel[in]- pointer to IIO channel structure
+ * @param id[in]- Attribute ID (optional)
+ * @return Number of characters read/written
+ */
+static int get_reconfigure_system(void *device,
+				  char *buf,
+				  uint32_t len,
+				  const struct iio_ch_info *channel,
+				  intptr_t id)
+{
+	return sprintf(buf, "%s", restart_iio_options[0]);
+}
+
+static int set_reconfigure_system(void *device,
```

Comment:
I think doxygen is added once for every getter/setter funcs together.. Is this intentional? I do understand that it's redundant to copy paste the same info 2 times everywhere!

Reply:
This is intentional. This is an old project and follows the older structure. Hence I retained the same for consistency